Makefile

```diff
@@ -68,6 +68,7 @@ docker:
 .PHONY: test
 test:
 	go test   ./test/... -test.v
+	go test ./controllers/... -test.v -ginkgo.v
```

Comment:
In my PR I've removed the prefix to just `./...`. Do we have a need to be more specific?

Reply:
For the controllers, I would prefer the option of `-ginkgo.v`

Reply:
If we add the option globally would it cause harm?

Reply:
Just tried doesn't work... 
On a side note: also if we use `./...` it traverses the whole project looking for the test files... making it a bit slower like for `api` `cmd`  `config` we don't have any test files there

Reply:
I'm planning to add a test to `pkg` as well. Let's see what merges and unpack it.